csync/src/std/asprintf.c

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
It's unlikely we'll update that code ever again so I think it's not worth it to start referencing files across the filesystem to 3rdparty in the cmake files.

Where should I put the URL, in the file itself?